test/unit/node/cli.test.ts

```diff
@@ -371,6 +371,9 @@ describe("parser", () => {
       }),
     ).toThrowError(expectedErrMsg)
   })
+  it("should not throw if an optional string is set to false", async () => {
+    expect(() => parse(["--cert=false"])).not.toThrowError()
```

Comment:
Should we check that it is not set in the result?  Something like:

```typescript
expect(parse(["--cert=false"])).toEqual({})
```

Reply:
Yeah, that's a better assertion IMO. Good callout!